jvm/sargon-android/src/main/java/com/radixdlt/sargon/extensions/WalletToDappInteractionResponse.kt

```diff
@@ -0,0 +1,13 @@
+package com.radixdlt.sargon.extensions
+
+import com.radixdlt.sargon.WalletToDappInteractionResponse
+import com.radixdlt.sargon.newWalletToDappInteractionResponseFromJsonBytes
+import com.radixdlt.sargon.walletToDappInteractionResponseToJsonBytes
+
+@Throws(SargonException::class)
+fun WalletToDappInteractionResponse.Companion.fromJson(json: String) =
+    newWalletToDappInteractionResponseFromJsonBytes(json.toByteArray().toBagOfBytes())
+
+@Throws(SargonException::class)
+fun WalletToDappInteractionResponse.toJson() =
+    walletToDappInteractionResponseToJsonBytes(this).string
```

Comment:
Also does this method throw @CyonAlexRDX ?

Reply:
To JSON never throws

Reply:
@jakub-rdx only mark Kotlin as throwing if the Rust code returns `Result`

Reply:
actually this one does not, at least it is not annotated in sargon.kt

Reply:
@micbakos-rdx updated